src/shell/shell.cpp

```diff
@@ -452,6 +453,67 @@ void DOS_Shell::SyntaxError()
 	WriteOut(MSG_Get("SHELL_SYNTAX_ERROR"));
 }
 
+static std::string get_cmd_history_path()
```

Comment:
`get_command_history_path` would be better.

Reply:
Changed